webpack-intro/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0",
+    "webpack": "^5.43.0",
+    "webpack-cli": "^4.7.2"
```

Comment:
webpack and webpack-cli should be installed as dev dependencies. You want to use `--save-dev`  to save a dependency as a dev dependency:

```
npm install webpack --save-dev
```

You can rerun the install command with `--save-dev` and they will be moved